pkg/beacon/relay/config/config.go

```diff
@@ -9,9 +9,6 @@ type Chain struct {
 	// HonestThreshold is the minimum number of active participants behaving
 	// according to the protocol needed to generate a new relay entry.
 	HonestThreshold int
-	// TicketSubmissionTimeout is the duration (in blocks) the staker has to
-	// submit any tickets to candidate to a new group.
-	TicketSubmissionTimeout uint64
```

Comment:
We can keep the number of rounds, round duration, and mining lag as constants (I wonder if it's possible to keep them in a library... 🤔) and reference them from config. This should be cheaper and consume less bytecode on the contract-side.

Reply:
`TicketSubmissionTimeout` will not be removed from `config.go`. Regarding other constants see [my comment above](https://github.com/keep-network/keep-core/pull/1510#discussion_r402240710).